server/csv-renderers/statistics-flow-operators.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var ensureObject     = require('es5-ext/object/valid-object')
+  , deferred         = require('deferred')
+  , modes            = require('../../utils/statistics-flow-group-modes')
+  , _                = require('mano').i18n.bind('Statistics operators csv')
+  , usersFullNameMap = require('../lib/user-full-names-map');
+
+module.exports = function (result, config) {
+	ensureObject(config);
+	var data, mode;
+	mode = modes.get(config.mode || 'daily');
+	data = [
+		[
+			mode.labelNoun,
```

Comment:
As per talk with @roxanadina this no longer applies: The CVS export should contain exactly the data shown on the page (same as PDF).